scripts/dev-e2e.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+STATUSFILE='armadactl-query.txt'
+AOREPO='https://github.com/armadaproject/armada-operator.git'
+AOHOME=../armada-operator
+ARMADA_CTL_VERSION='0.18.3'
+TEST_QUEUE='test'
+JOB_DETAILS=1
+
+usage() {
+  cat > /dev/stderr  <<USAGE
+Usage: ${0} -s
+  -s => get job status details from Armada
+
+USAGE
+  exit 1
+}
+
+while getopts "s" opt; do
+    case "${opt}" in
+        s )
+          JOB_DETAILS=1 ;;
+        * )
+          echo "opt is $opt"
+          usage ;;
+    esac
+done
+
+scripts="$(cd "$(dirname "$0")" || exit; pwd)"
+source "$scripts"/init.sh
+
+log() {
+    echo -e "${GREEN}$1${NC}"
```

Comment:
I don't think all rows should be green, those that stress something succeeded should be green, those communicate failure should be red. All others without color.

Reply:
Errors are now rendered with red prefix;  green message prefixes are greatly reduced - now just for notable successes - in commit 433fa26.